h2o-py/tests/testdir_algos/gam/pyunit_PUBDEV_7931_gam_null_predictors.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import division
+from __future__ import print_function
+import sys
+sys.path.insert(1, "../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.gam import H2OGeneralizedAdditiveEstimator
+
+# In this test, we check and make sure Gam can run without predictor columns as long as gam column is specified
+def test_gam_null_predictors():
+    print("Checking null predictor run for binomial")
+    h2o_data = h2o.import_file(pyunit_utils.locate("smalldata/gam_test/gamBinomial1Col.csv"))
+    myY = "response"
+    buildModelMetricsCheck(h2o_data, 'binomial')
+
+    print("Checking null predictor for gaussian")
```

Comment:
out of curiosity, whats the value for testing different family?

Reply:
The coefficient assignment between binomial and multinomial is different.  But you are right, Gaussian is the same as binomial.  Hence, that test is probably redundant.